outputs/alertmanager.go

```diff
@@ -11,6 +12,18 @@ const (
 	AlertmanagerURI string = "/api/v1/alerts"
 )
 
+var priorityMap = map[string]int{
```

Comment:
you do not need to declare this variable, the `priorityMap` is already a global variable, see : https://github.com/falcosecurity/falcosidekick/blob/master/main.go#L37

Reply:
ok will fix MR